server/channels/app/session.go

```diff
@@ -205,7 +207,9 @@ func (a *App) RevokeAllSessions(c request.CTX, userID string) *model.AppError {
 }
 
 func (a *App) AddSessionToCache(session *model.Session) {
-	a.ch.srv.platform.AddSessionToCache(session)
+	if appErr := a.ch.srv.platform.AddSessionToCache(session); appErr != nil {
+		a.Srv().Platform().Log().Error("Failed to add session to cache", mlog.Err(appErr), mlog.String("session_id", session.Id), mlog.String("user_id", session.UserId))
+	}
```

Comment:
That is not an `appErr`
```suggestion
	if err := a.ch.srv.platform.AddSessionToCache(session); err != nil {
		a.Srv().Platform().Log().Error("Failed to add session to cache", mlog.String("session_id", session.Id), mlog.String("user_id", session.UserId), mlog.Err(err))
	}
```